lib/experimental/script-modules.php

```diff
@@ -218,12 +255,8 @@ function gutenberg_register_script_modules() {
 		array(),
 		$default_version
 	);
-	add_filter(
-		'script_module_data_@wordpress/a11y',
-		function ( $data ) {
-			$data['i18n'] = array( 'Notifications' => __( 'Notifications', 'default' ) );
-			return $data;
```

Comment:
No more ugly "localization" for this module.

Reply:
Right, that's a nice enhancement. However, printing the entire HTML on the server is less optimal than with JS. Can CSS be moved to inline `style`, for example? I guess, it was all put in JS to make it a standalone package. Overall, @afercia might be the best person to advice here how to make it usable as ES module.

Reply:
I was mostly saying that this HTML can be further optimized, which you did already by wrapping it with a div that contains inline styles that hide it from view. I'm also wondering if we really need two distinct aria live regions, as I was reading that it can get changed also dynamically. That can be explored later.

Reply:
> I'm also wondering if we really need two distinct aria live regions

Yes they are needed. Changing polite/assertive on the fly may not work reliably as screen readers need to _monitor_ these elements.

Reply:
Works for me. Thank you, @afercia, for clarifying this one.